src/styles/blocks/body.scss

```diff
@@ -0,0 +1,8 @@
+.body {
+  margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  //@include centering-xy();
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments)